tools/ci/packaging/archlinux/build.sh

```diff
@@ -15,6 +15,7 @@ sed -i "s|/A3ULPATH|$A3UL_DIR|g" PKGBUILD
 
 mkdir src
 cp -RL /$A3UL_DIR/* ./src
+chown -R builduser:builduser .
```

Comment:
:thinking: Maybe you can try to use:
`chown -R builduser:builduser "$SELF_PATH"`

It will do the same but doesn't use the dot :wink: